2d/platformer/src/titlescreen/TitleScreen.tscn

```diff
@@ -0,0 +1,66 @@
+[gd_scene load_steps=8 format=2]
+
+[ext_resource path="res://src/titlescreen/TitleScreen.gd" type="Script" id=1]
+[ext_resource path="res://assets/art/background/sky.png" type="Texture" id=2]
+[ext_resource path="res://assets/art/titlescreen/titletogame.png" type="Texture" id=3]
+[ext_resource path="res://assets/art/titlescreen/starbuttonselected.png" type="Texture" id=4]
+[ext_resource path="res://assets/art/titlescreen/startbutton.png" type="Texture" id=5]
+[ext_resource path="res://assets/art/titlescreen/exitselected.png" type="Texture" id=6]
+[ext_resource path="res://assets/art/titlescreen/exitbutton.png" type="Texture" id=7]
+
+[node name="TitleScreen" type="Node"]
+script = ExtResource( 1 )
+
+[node name="Sprite" type="Sprite" parent="."]
+position = Vector2( 3.69324, 6.13867 )
+scale = Vector2( 49.6504, 1.75295 )
+texture = ExtResource( 2 )
+centered = false
+
+[node name="MarginContainer" type="MarginContainer" parent="."]
+anchor_right = 1.0
+anchor_bottom = 1.0
+margin_left = -1.22774
+margin_top = 2.45547
+margin_right = -1.22772
+margin_bottom = 20.4555
+custom_constants/margin_top = 20
+custom_constants/margin_bottom = 20
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="VBoxContainer" type="VBoxContainer" parent="MarginContainer"]
+margin_left = 49.0
+margin_top = 20.0
+margin_right = 751.0
+margin_bottom = 478.0
+size_flags_horizontal = 4
+custom_constants/separation = 16
+
+[node name="TextureRect" type="TextureRect" parent="MarginContainer/VBoxContainer"]
+margin_right = 702.0
+margin_bottom = 155.0
+texture = ExtResource( 3 )
+
+[node name="VBoxContainer" type="VBoxContainer" parent="MarginContainer/VBoxContainer"]
+margin_left = 266.0
+margin_top = 171.0
+margin_right = 435.0
+margin_bottom = 329.0
+size_flags_horizontal = 4
+
+[node name="TextureButton" type="TextureButton" parent="MarginContainer/VBoxContainer/VBoxContainer"]
+margin_right = 169.0
+margin_bottom = 76.0
+texture_normal = ExtResource( 5 )
+texture_focused = ExtResource( 4 )
+
+[node name="TextureButton2" type="TextureButton" parent="MarginContainer/VBoxContainer/VBoxContainer"]
+margin_top = 80.0
+margin_right = 169.0
+margin_bottom = 158.0
+texture_normal = ExtResource( 7 )
+texture_focused = ExtResource( 6 )
+[connection signal="pressed" from="MarginContainer/VBoxContainer/VBoxContainer/TextureButton" to="." method="_on_TextureButton_pressed"]
+[connection signal="pressed" from="MarginContainer/VBoxContainer/VBoxContainer/TextureButton2" to="." method="_on_TextureButton2_pressed"]
```

Comment:
These buttons should be named something else. Something to give an indication about what they are or what they do.

Reply:
I would recommend renaming the Nodes to reflect their "parents"

Reply:
I agree with the previous comments. Your variable and object names should match a consistent pattern throughout the project, for the sake of readability.